README.md

```diff
@@ -1,11 +1,13 @@
-# PILZ Safety Laserscanner PSENscan
+[![PILZ](img/pilz-logo.png)](https://www.pilz.com)
+
+# PILZ Safety laser scanner PSENscan
 
 ## Package: psen_scan
 
-The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the Laserscanner-data into your ROS-Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][].
+The **psen_scan** package is a ROS integration of the PSENscan Safety laser scanner product. It allows for an easy integration of the laser scanner data into your ROS Environment. Using the standard [sensor_msgs/LaserScan][] message format ensures compatibility with other laserscan-post-processing nodes such as [gmapping][]. For a general overview and link collection we refer to the [wiki page](http://wiki.ros.org/psen_scan).
 
 <p align="center">
-<img src="img/PSENscan.jpg" alt="Pilz Safety Laserscanner" title="Pilz Safety Laserscanner">
+<img src="img/PSENscan.jpg" alt="Pilz Safety laser scanner" title="Pilz Safety laser scanner">
```

Comment:
maybe adjust to uppercase from heading?
```suggestion
<img src="img/PSENscan.jpg" alt="PILZ Safety laser scanner" title="PILZ Safety laser scanner">
```

Reply:
Solved in 01dcea65f79ebae8a82975a9271cb7ec29a87550